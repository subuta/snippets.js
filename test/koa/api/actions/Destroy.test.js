/* global expect, describe, it */

import { build, format, snippets as s } from 'bld.js'

import DestroyAction from 'lib/koa/api/actions/Destroy'

describe('koa/api/actions/Destroy', () => {
  it('should Destroy koa action', () => {
    expect(format(DestroyAction({
      model: 'user',
      routeConfig: {}
    }))).toMatchSnapshot()
  })
})
