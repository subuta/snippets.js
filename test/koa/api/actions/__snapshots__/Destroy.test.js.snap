// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`koa/api/actions/Destroy should Destroy koa action 1`] = `
"user.delete('/:id', auth, async (ctx) => {
  const {User} = ctx.state.models
  let params = {id: ctx.params.id}

  /* mat Before destroy [start] */
  /* mat Before destroy [end] */

  await User.query()
    .delete()
    .where(params)
  ctx.body = null
})
"
`;
